tests/test_relations.py

```diff
@@ -13,6 +13,14 @@ def to_representation(self, instance):
         return {"pk": instance.pk, "name": instance.name}
 
 
+class RecursiveSerializer(serializers.Serializer):
+    pk = serializers.CharField()
+    recursive_field = PresentablePrimaryKeyRelatedField(
```

Comment:
To test many=True, you can add a field called `recursive_fields` with many=True.

and your self.related_object will become something like below.
```

        self.related_object = MockObject(
            pk=3,
            name="baz",
            recursive_fields = [mock object, mock object],
            recursive_field=MockObject(
                pk=4,
                name="foobar",
                recursive_fields=[],
                recursive_field=MockObject(
                    pk=5,
                    name="barbaz",
                    recursive_fields=[],
                    recursive_field=None)
            ),
        )
```

Reply:
Could you please pull the master to fix conflicts?

Reply:
Done :+1: